src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/event_dispatcher_1_event1.txt

```diff
@@ -1,8 +1,8 @@
 <info>[event_dispatcher]</info> Registered listeners for event <info>event1</info>
 
-+-------+-------------------+
-| Order | Callable          |
-+-------+-------------------+
-| #1    | global_function() |
-| #2    | \Closure()        |
-+-------+-------------------+
++-------+-------------------+----------+
+| Order | Callable          | Priority |
++-------+-------------------+----------+
```

Comment:
Yes I wasn't quite sure why the Order column was there, maybe for people who can't count :P If you'd like to send a PR to remove it from the TextDescriptor that sounds good to me.